pkg/providers/launchtemplate/launchtemplate_test.go

```diff
@@ -674,7 +844,7 @@ var _ = Describe("LaunchTemplates", func() {
 		var info *ec2.InstanceTypeInfo
 		BeforeEach(func() {
 			var ok bool
-			instanceInfo, err := instanceTypeProvider.getInstanceTypes(ctx)
+			instanceInfo, err := instanceTypeProvider.GetInstanceTypes(ctx)
```

Comment:
Rather than exposing this through the instanceTypeProvider, can we just go to the fake EC2 API for this test case?